test_common/KokkosKernels_TestUtils.hpp

```diff
@@ -661,8 +670,8 @@ class RandCsMatrix {
 
     info = std::string(
         std::string("RandCsMatrix<") + typeid(ScalarType).name() + ", " +
-        typeid(LayoutType).name() + ", " + typeid(ExeSpaceType).name() + ">(" +
-        std::to_string(dim2) + ", " + std::to_string(dim1) +
+        typeid(LayoutType).name() + ", " + typeid(execution_space).name() +
```

Comment:
```suggestion
        typeid(LayoutType).name() + ", " + execution_space().name() +
```
for a non-mangled exec space name